bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: openshift-kube-scheduler
   labels:
     app: openshift-kube-scheduler
+    deployment-id: "DEPLOYMENT_ID"
```

Comment:
we updated this to REVISION, compare @deads2k's recent library-go PR.

Reply:
@sttts thanks, will update soon.